src/Features/Core/Portable/GenerateMember/GenerateConstructor/AbstractGenerateConstructorService.Editor.cs

```diff
@@ -79,30 +82,36 @@ private partial class Editor
 
             internal async Task<(Document, bool addedFields)> GetEditAsync()
             {
+                // Get naming rule for generating fields
+                var rules = await _document.Document.GetNamingRulesAsync(
+                    FallbackNamingRules.RefactoringMatchLookupRules, _cancellationToken).ConfigureAwait(false);
+                var fieldNamingRule = rules.Where(c => c.SymbolSpecification.AppliesTo(new SymbolKindOrTypeKind(SymbolKind.Field),
+                    DeclarationModifiers.None, Accessibility.Private)).First();
```

Comment:
how do we know that 'First' will work here?

Reply:
It's the same as this: 

https://github.com/dotnet/roslyn/blob/0f5f87379090fbe906fb6c27209d970dd0175321/src/Features/Core/Portable/InitializeParameter/AbstractInitializeMemberFromParameterCodeRefactoringProviderMemberCreation.cs#L139-L141